lib/run.js

```diff
@@ -57,7 +57,7 @@ class HelmRenderEngine extends RenderEngine {
             yield this.isHelmV3(helmPath).then(() => { isV3 = true; }).catch(() => { isV3 = false; });
             try {
                 if (!isV3) {
-                    yield utilities.execCommand(helmPath, ['init', '--client-only'], options);
+                    yield utilities.execCommand(helmPath, ['init', '--client-only', '--stable-repo-url', 'https://charts.helm.sh/stable'], options);
```

Comment:
Is this only a v3 thing

Reply:
Yes, helm init is used for tiller which is supported in v2 only.